### DISCUSSION TITLE
Guidelines for anti-malware exclusion list

### DISCUSSION THREAD

**Discussion by jbcsc** [concourse/concourse#7215] [Help & Support]
Hello all,

We have the ds_agent ( Deep security agent) running on the worker nodes. However, the scan is eating a lot of CPU and slowing down the builds a lot. As example, without ds_agent running the build takes 90s, when it is running it takes 12m.

Do you have any guidelines of what should be excluded (paths/processes/etc) for anti-malware software scans?
In special for the dsa?
https://success.trendmicro.com/solution/1122045-adding-exclusion-for-anti-malware-real-time-scan-in-deep-security

Thanks,
João

**Comment by taylorsilva** [comment]
I don't have any suggestion for any paths to exclude. I think a hard thing about this is that Concourse workers are constantly creating and deleting paths. I can image from the scanners point of view it's an endless game of wac-a-mole, with paths appearing and disappearing constantly. I'm not sure how smart this scanner is with scanning overlay mounts, but if it's not very smart then it could be scanning the same files multiple times as it looks through the upper, lower, merged, and working dirs of a single overlay volume.
